.github/actions/check-pr/package-lock.json

```diff
@@ -0,0 +1,431 @@
+{
+  "name": "check-pr",
```

Comment:
[q] Why is this a `package-lock.json` and not `yarn.lock`?

Reply:
I just did the same thing as with the other action. I think yarn might not be supported in GH actions, but I didn't want to try on this PR...

Reply:
I'm fine with npm, just wanted to mention that its possible to use yarn for actions, I've done so on my [personal open source project](https://github.com/fwilhe2/setup-kotlin)

Reply:
I guess, if we are using yarn it makes sense to do this here as well. Still, I prefer to do this in a separate PR to adjust the other action that we have as well. I created a ticket to checkout ncc, I will add this as well.

Reply:
=> https://github.com/SAP/cloud-sdk-backlog/issues/995